Source/Scene/ModelExperimental/buildDrawCommands.js

```diff
@@ -63,7 +77,7 @@ export default function buildDrawCommands(
     pickId: primitiveRenderResources.pickId,
     instanceCount: primitiveRenderResources.instanceCount,
     primitiveType: primitiveRenderResources.primitiveType,
-    debugShowBoundingVolume: model.debugShowBoundingVolume,
+    debugShowBoundingVolume: true,
```

Comment:
Remember to revert this line.

![image](https://user-images.githubusercontent.com/8422414/148570621-ba1b3023-5d64-45cc-b48d-d7870f3b8c33.png)

For individual models, this can be set from the `fromGltf` method, but I do wish we had a way to control this for tilesets. (e.g. `tileset.debugShowBoundingSpheres`). I'll open an issue because this is likely somewhat involved (as it would apply to more than `ModelExperimental`)

Reply:
opened https://github.com/CesiumGS/cesium/issues/9996 for the more general flag